frontend/src/App.tsx

```diff
@@ -38,12 +41,21 @@ function App() {
   const handle = match?.handle as Record<string, unknown>;
 
   const [user, setUser] = useState<User | null>(null);
+  const [oidcInfo, setOidcInfo] = useState<OidcInfo>({
```

Comment:
I think both the `whoAmI` and the `getOIDCInfo` can be move to `app.loader.ts` and consumed using the `useLoaderData` hook.

Reply:
Made an attemtp, but failed because of circular dependencies :disappointed: